net-im/skypeforlinux/skypeforlinux-9999.ebuild

```diff
@@ -93,8 +91,9 @@ src_install() {
 #	insinto /etc/dbus-1/system.d
 #	doins ${PN}.conf
 
-	dodoc usr/share/doc/skypeforlinux/* usr/share/skypeforlinux/*.html
-	dodoc usr/share/skypeforlinux/*.txt usr/share/skypeforlinux/LICENSE
+	dodoc usr/share/doc/skypeforlinux/*
+	dodoc usr/share/skypeforlinux/LICENSE
+	dodoc usr/share/skypeforlinux/LICENSES.chromium.html
 
 	# create compat symlink
 	dosym ${P} /usr/share/doc/skypeforlinux
```

Comment:
Previous version of the ebuild made a symlink from skypeforlinux-<version> to just plain skypeforlinux.

Its not upstream hardcoded, but this is how the previous edition worked.

Is this just junk left behind by the previous maintainer?